dockerfiles/cpubuilder_ubuntu_jammy_ghr_x86_64.Dockerfile

```diff
@@ -37,6 +49,7 @@ RUN ln -s /usr/bin/lld-14 /usr/bin/lld && \
     ln -s /usr/bin/ld.lld-14 /usr/bin/ld.lld && \
     ln -s /usr/bin/clang-14 /usr/bin/clang && \
     ln -s /usr/bin/clang++-14 /usr/bin/clang++
+# Default to using clang. This can be overriden to gcc as desired.
```

Comment:
Drop extra comment. That's already covered above:
```
# Setup symlinks and alternatives then default to using clang-14.
# This can be overriden to gcc or another clang version as needed.
```

(I guess I had a typo in there, - should be "overridden" could fix or ignore)